docs/content/4.data/1.table.md

```diff
@@ -524,6 +524,66 @@ excludedProps:
 ---
 ::
 
+## Styling
+
+You can apply styles to `tr` and `td` elements by passing a `class` to rows.
+
+Also, you can apply styles to `th` elements by passing a `class` to columns.
```

Comment:
Additionally, I have included instructions on how to format columns.

Reply:
Thank you for this